www/userguide/userguide.md

```diff
@@ -0,0 +1,254 @@
+- [Report navigation tree](#userguide_report_navigation_tree)
+- [Button pane](#userguide_button_pane)
+- [Bug path view](#userguide_bug_path_view)
+- [Comments](#userguide_comment)
+
+![Report view](images/userguide/report_view.png)
+
+\subsection userguide_report_navigation_tree Report navigation tree
+Report Navigation Tree shows the found reports at the currently opened file. The
+reports are grouped by the severity level.
+You can navigate between them by clicking on a node in the tree.
+
+![Report navigation tree](images/userguide/report_navigation_tree.png)
+
+\subsection userguide_button_pane Button pane
+Button Pane contains several items which help you to change or get some
+property of the currently opened report. 
+![Button pane](images/userguide/button_pane.png)
+
+\subsubsection userguide_checker_documentation Show documentation
+Show Documentation button shows the documentation of the actual checker which
+identified by the currently opened report.
+![Checker documentation](images/userguide/checker_documentation.png)
+
+\subsubsection userguide_review_status Review status
+Reports can be assigned a review status of Unreviewed, Confirmed bug, False
+positive, Intentional, along with an optional comment on why this status was
+applied.
+Icon  | Status | Meaning
+:-------------: | ------------- | -------------
+<span class="customIcon review-status-unreviewed"></span>  | Unreviewed (default) | Nobody has seen this report |
+<span class="customIcon review-status-confirmed-bug"></span>  | Confirmed | This is really a bug |
+<span class="customIcon review-status-false-positive"></span>  | False positive | This is not a bug |
+<span class="customIcon review-status-intentional"></span>  | Intentional | This report is a bug but we don't want to fix it |
+
+We can change the review status from the default
+<span class="customIcon review-status-unreviewed"></span> _Unreviewed_ option
+to something else in the report details view above the file view.
+![Unreviewed](images/userguide/review_status_unreviewed.png)
+
+If you changed the review status, you can optionally explain the reason why
+you changed it.
+![Change review status](images/userguide/review_status_change.png)
+
+If somebody has already changed the review status from the default one, you can
+see extra information (who changed the review status, when and why) beside
+the review status selector by hovering on the message icon. This message icon is
+hidden by default if nobody has changed the review status.
+![Review status message](images/userguide/review_status_message.png)
+
+\subsection userguide_same_reports Same reports
+Several reports may belong to a specific bug if the but itself can be reached on
+different control paths. In the Bug viewer you can check whether the selected
+bug is available on a different path.
+
+![Same reports](images/userguide/same_reports.png)
+
+\subsection userguide_bug_path_view Bug path view
+Bug path shows the path of the actual report.
```

Comment:
"Some checkers are able to follow the execution path along the control flow of the program. If a bug appears on any of these paths then CodeChecker is able to present the full path on which this so called symbolic execution reached the place of error. This path can be checked in this bug path view."